src/intervals/conversion.jl

```diff
@@ -104,7 +104,8 @@ end
 function atomic(::Type{Interval{T}}, x::S) where {T<:AbstractFloat, S<:AbstractFloat}
     isinf(x) && return wideinterval(T(x))
 
-    xrat = rationalize(x)
+    # TODO: Delegate type assertion to the type system
+    xrat = T == BigFloat ? rationalize(BigInt, x) : rationalize(x) # Issue #410
```

Comment:
Please change this to use dispatch with appropriate methods.

Reply:
Done! PR updated with (i) more specialized

```atomic(::Type{Interval{BigFloat}}, x::S) where {S<:AbstractFloat}```

method definition and (ii) tests for the `±` and `..` interval construction operators with `BigFloat` precision arguments (using lowercase variables only).